### PR TITLE
Prepared for PHP 7.4 update

### PR DESCRIPTION
**Pull request by olexiyk**
Switched to github actions, removed composer.lock from repo, ran PHP-CS-Fixer migrations up to PHP 7.3

### CODE REVIEW

**Comment by olexiyk** [comment]
Github actions https://github.com/olexiyk/converter/actions

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+on: [ push, pull_request ]
+
+jobs:
+  build-test:
+    name: Test on PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: [ '7.3', '7.4' ]
```
Comment (nawarian) [inline .github/workflows/ci.yaml]: Does it make sense to add php 8.0 here too?

**Review by nawarian — APPROVED** [review]
LGTM

Thanks a lot for the effort!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (olexiyk) [reply .github/workflows/ci.yaml]: Yes, thanks for a suggestion

**Pull request merged**